platform/kube/conversion.go

```diff
@@ -53,11 +55,29 @@ func convertLabels(obj meta_v1.ObjectMeta) model.Labels {
 	return out
 }
 
-func convertPort(port v1.ServicePort) *model.Port {
+// Extracts security option for given port from annotation. If there is no such
+// annotation, or the annotation value is not recognized, returns
+// model.SecurityDefault.
+func extractAuthenticationPolicy(port v1.ServicePort, obj meta_v1.ObjectMeta) model.AuthenticationPolicy {
```

Comment:
Can you also extract from namespace level? It might become cumbersome to annotate every service's port. And for system managed namespaces / administrative namespaces (e.g., kube-system, calico-system, ibm-system), we need an exception mechanism for the entire namespace.

Reply:
From user point of view, I think we need this fine grained control (e.g they may need to turn of TLS for a some service which already have TLS, eg. DB). Ask them to move the job to different namespace for that purpose seems unnatural.

We may added a config at namespace level later, if there are some demands. Or at least add "wild-card" to this to mach to all ports.

Reply:
We will definitely need to annotate hierarchically across istio, for auth, the automatic injection, and anything else.

But it is ok to do it incrementally and start with services.

Reply:
We need per-service for sure and first - so individual services can be moved incrementally. 
We do need namespace level as well - but I think it is better to keep PRs small and have incremental changes.